src/sentry/integrations/github/integration.py

```diff
@@ -76,6 +80,7 @@ class GitHubIntegrationProvider(IntegrationProvider):
     name = 'GitHub'
     metadata = metadata
     integration_cls = GitHubIntegration
+    features = frozenset([IntegrationFeatures.ISSUE_SYNC])
```

Comment:
so I either think we want to make a separate feature for the non-syncing ones or override the base class's methods like `sync_assignee_outbound` with just a `pass`.

@EvanPurkhiser do you have opinions? just having a different feature might be better if we're going to possibly use that in some config page that describes what features the integration has.

Reply:
I think two features probably would make sense.